lib/recurly/addon.php

```diff
@@ -1,5 +1,24 @@
 <?php
 
+/**
+ * class Recurly_Addon
+ * @property Recurly_Stub $plan The URL of the associated Recurly_Plan for this addon.
+ * @property Recurly_Stub $measured_unit The URL of the associated Recurly_MeasuredUnit for this addon.
+ * @property string $add_on_code The add-on code. Max of 50 characters.
+ * @property string $name The add-on name. Max of 255 characters.
+ * @property int $default_quantity Default quantity for the hosted pages, defaults to 1.
+ * @property int $default_quantity_on_hosted_page If true, displays a quantity field on the hosted pages for the add-on.
+ * @property string $tax_code Optional field for EU VAT merchants and Avalara AvaTax Pro merchants. If you are using Recurly's EU VAT feature, you can use values of 'unknown', 'physical', or 'digital'. If you have your own AvaTax account configured, you can use Avalara's tax codes to assign custom tax rules.
```

Comment:
We've got a few styles of listing to valid enum values: 
- `Allowed values: [referral,`...
- `values of 'unknown', 'physical',`...
- `is Fixed-Price (fixed), or Usage-Based (usage)`

It would be nice to standardize it a bit. Are we going to quote the values? etc